api/tests/integration/infrastructure/utils/ods/read-ods-utils_test.js

```diff
@@ -223,13 +221,11 @@ describe('Integration | Infrastructure | Utils | Ods | read-ods-utils', function
         odsBuffer = await readFile(NEW_LINE_ODS_FILE_PATH);
 
         // when
+        // then
         await validateOdsHeaders({
           odsBuffer,
           headers: VALID_HEADERS,
         });
```

Comment:
Ca peut être quelque chose comme ça aussi : 
```suggestion
        const call = () => {
          validateOdsHeaders({
            odsBuffer,
            headers: VALID_HEADERS,
          });
        };

        // then
        expect(call).to.not.throw();
```

Reply:
Très juste, corrigé